GUI/src/main/java/backend/FileDownloader.java

```diff
@@ -113,19 +113,22 @@ protected Integer call() {
         updateProgress(0.0, 1.0);
         done = true;
         return exitCode;
+        //no extra space
     }
 
     private void downloadYoutubeOrInstagram(boolean isSpotifySong) {
         String[] fullCommand = new String[]{YT_DLP, "--quiet", "--progress", "-P", dir, downloadLink, "-o", filename, "-f", (isSpotifySong ? "bestaudio" : "mp4")};
         ProcessBuilder processBuilder = new ProcessBuilder(fullCommand);
         sendInfoMessage(String.format(DOWNLOADING_F, filename));
         Process process = null;
+        int exitCode = -1;
```

Comment:
I don't think this initialisation of the local `exitCode` variable is required. An `exitCode` object variable is already declared at [line 50](https://github.com/SaptarshiSarkar12/Drifty/blob/725113cab301a48f970817bb02ea6e6e21150af7/GUI/src/main/java/backend/FileDownloader.java#L50) and initialised to **1** by default (a standard generic error code).
Could you please remove this line?